### PR TITLE
replace stream

### PR DESCRIPTION
**Pull request by fershady**

### CODE REVIEW

**Review by Dum4G — DISMISSED** [review]

File: channels/cr.m3u [hunk 1]
```diff
@@ -11,6 +11,8 @@ https://rtmp.info/anexiontv/envivo/playlist.m3u8
 http://inliveserver.com:1935/14510/14510/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal8.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://image.multimedios.cr/sites/default/files/styles/social_share/public/multimedios1.jpg" group-title="",Canal 8 (720p)
 http://mdstrm.com/live-stream-playlist/5a7b1e63a8da282c34d65445.m3u8
+#EXTINF:-1 tvg-id="Canal8.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://es.wikipedia.org/wiki/Canal_11_(Costa_Rica)#/media/Archivo:Repretel_11_logo.png" group-title="General",Canal 11
```
Comment (Dum4G) [inline channels/cr.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="Canal11.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://es.wikipedia.org/wiki/Canal_11_(Costa_Rica)#/media/Archivo:Repretel_11_logo.png" group-title="General",Canal 11
```

**Review by Dum4G — DISMISSED** [review]

File: channels/cr.m3u [hunk 2]
```diff
@@ -11,6 +11,8 @@ https://rtmp.info/anexiontv/envivo/playlist.m3u8
 http://inliveserver.com:1935/14510/14510/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal8.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://image.multimedios.cr/sites/default/files/styles/social_share/public/multimedios1.jpg" group-title="",Canal 8 (720p)
 http://mdstrm.com/live-stream-playlist/5a7b1e63a8da282c34d65445.m3u8
+#EXTINF:-1 tvg-id="Canal11.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://es.wikipedia.org/wiki/Canal_11_(Costa_Rica)#/media/Archivo:Repretel_11_logo.png" group-title="General",Canal 11
+https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/chunklist_b1248000.m3u8
```
Comment (LaneSh4d0w) [inline channels/cr.m3u:15]:
```suggestion
https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/playlist.m3u8
```

Adaptive

File: channels/mx.m3u [hunk 3]
```diff
@@ -56,7 +56,7 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtu
 #EXTINF:-1 tvg-id="ImagenRadio.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SINYy29.png" group-title="",Imagen Radio (720p) [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCB0BUmdBOrH9mYU2ebs1eWA/live
 #EXTINF:-1 tvg-id="Imagentv.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/mRGetOJ.png" group-title="",Imagen TV (720p) [Geo-blocked]
-https://stream-04.nyc.dailymotion.com/sec(YDiRVmci6dFcTAVfdXnSLkVhTJz_zPGOO7WKknz-Hj0)/dm/3/x82z4if/s/live-2.m3u8
+https://stream-01.nyc.dailymotion.com/sec(YDiRVmci6dFcTAVfdXnSLrUbmjbCrTRoUsPvrxQgIb0)/dm/3/x82z4if/d/live-2.m3u8
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:59]:
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x82z4if ?
```

Comment (Dum4G) [reply channels/mx.m3u:59]: Actually dodged geo-block for me too

File: channels/mx.m3u [hunk 4]
```diff
@@ -56,7 +56,7 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtu
 #EXTINF:-1 tvg-id="ImagenRadio.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SINYy29.png" group-title="",Imagen Radio (720p) [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCB0BUmdBOrH9mYU2ebs1eWA/live
 #EXTINF:-1 tvg-id="Imagentv.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/mRGetOJ.png" group-title="",Imagen TV (720p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/mx.m3u:58]:
```suggestion
#EXTINF:-1 tvg-id="Imagentv.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/mRGetOJ.png" group-title="",Imagen TV (720p)
```

**Review by Dum4G — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Little bonus : https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal2/smil:live.smil/playlist.m3u8 Repretel CDR-2 Central de Radios

**Review by BellezaEmporium — DISMISSED** [review]

File: channels/cr.m3u [hunk 5]
```diff
@@ -9,8 +9,12 @@ http://rtmp.info:8081/anexiontv/envivo/index.m3u8
 https://rtmp.info/anexiontv/envivo/playlist.m3u8
 #EXTINF:-1 tvg-id="AntenaSeisTV.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://lh3.googleusercontent.com/V-_fSNpzlQPfFuUG0wLkrR7-PQ3RDx3n9tE4zA_8Exm_OKVeU24Kt2HiKLYweJXItgY" group-title="",Antena Seis TV (720p) [Geo-blocked]
 http://inliveserver.com:1935/14510/14510/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal6.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="" group-title="",Canal 6 (720p)
+https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal6/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal8.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://image.multimedios.cr/sites/default/files/styles/social_share/public/multimedios1.jpg" group-title="",Canal 8 (720p)
 http://mdstrm.com/live-stream-playlist/5a7b1e63a8da282c34d65445.m3u8
+#EXTINF:-1 tvg-id="Canal11.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://es.wikipedia.org/wiki/Canal_11_(Costa_Rica)#/media/Archivo:Repretel_11_logo.png" group-title="General",Canal 11
+https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/playlist.m3u8
```
Comment (Dum4G) [inline channels/cr.m3u:17]:
```suggestion
https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/playlist.m3u8
#EXTINF:-1 tvg-id="Canal2.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="" group-title="General",Repretel CDR-2 Central de Radios
https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal2/smil:live.smil/playlist.m3u8
```

**Review by Dum4G — APPROVED** [review]

File: channels/cr.m3u (hunk 5 above)
Comment (LaneSh4d0w) [reply channels/cr.m3u:17]: It's a radio, hence the reason why I did not add it.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by Johnrebbel** [comment]
Had to Uninstall Android auto and then I installed 7.0 version. Everything on aastore works again

**Pull request merged**